clients/client-python/gravitino/client/gravitino_client_base.py

```diff
@@ -75,7 +75,7 @@ def load_metalake(self, name: str) -> GravitinoMetalake:
 
         self.check_metalake_name(name)
         response = self._rest_client.get(
-            GravitinoClientBase.API_METALAKES_IDENTIFIER_PATH + name
+            GravitinoClientBase.API_METALAKES_IDENTIFIER_PATH + name,
```

Comment:
What is the purpose of this change?

Reply:
Oh I add some arguments here when I tested, I will fix it.

Reply:
Fixed.